config/default.js

```diff
@@ -17,7 +17,8 @@ module.exports = {
     AWS_WRITE_UNITS: process.env.AWS_WRITE_UNITS || 5,
     S3_BUCKET: process.env.S3_BUCKET || 'tc-testing-submissions' // S3 Bucket to which submissions need to be uploaded
   },
-  BUSAPI_EVENTS_URL: process.env.BUSAPI_EVENTS_URL || 'https://api.topcoder-dev.com/v5/bus/events',
```

Comment:
Why we removed /bus/events here? is it intentional or by mistake?

Reply:
@BadalZ We have started using Bus API wrapper instead of calling Bus API directly, Bus API wrapper will have the suffix URLs

Reply:
thanks for clarification